ui/raidboss/data/06-ew/trial/hydaelyn-ex.txt

```diff
@@ -152,54 +168,57 @@ hideall "--sync--"
 2026.9 "Mousa's Scorn" sync / 1[56]:[^:]*:Hydaelyn:65C0:/
 2032.1 "--middle--" sync / 1[56]:[^:]*:Hydaelyn:6E7A:/
 2038.7 "Parhelion" sync / 1[56]:[^:]*:Hydaelyn:65B0:/
-2039.0 "Beacon 1" sync / 1[56]:[^:]*:Parhelion:65B2:/
-2040.6 "Beacon 2" sync / 1[56]:[^:]*:Parhelion:65B2:/
-2042.2 "Beacon 3" sync / 1[56]:[^:]*:Parhelion:65B2:/
+2039.0 "Beacon 1" #sync / 1[56]:[^:]*:Parhelion:65B2:/
+2040.6 "Beacon 2" #sync / 1[56]:[^:]*:Parhelion:65B2:/
+2042.2 "Beacon 3" #sync / 1[56]:[^:]*:Parhelion:65B2:/
 2047.5 "Crystallize" sync / 1[56]:[^:]*:Hydaelyn:6ED5:/
 2055.6 "Subparhelion" sync / 1[56]:[^:]*:Hydaelyn:6C56:/
 2057.6 "Crystalline Water III" sync / 1[56]:[^:]*:Hydaelyn:65A2:/
-2057.8 "Beacon 1" sync / 1[56]:[^:]*:Parhelion:65B3:/
+2057.8 "Beacon 1" #sync / 1[56]:[^:]*:Parhelion:65B3:/
 2058.7 "--sync--" sync / 1[56]:[^:]*:Hydaelyn:65A1:/
-2059.4 "Beacon 2" sync / 1[56]:[^:]*:Parhelion:65B3:/
-2061.0 "Beacon 3" sync / 1[56]:[^:]*:Parhelion:65B3:/
+2059.4 "Beacon 2" #sync / 1[56]:[^:]*:Parhelion:65B3:/
+2061.0 "Beacon 3" #sync / 1[56]:[^:]*:Parhelion:65B3:/
 2065.8 "Aureole/Lateral Aureole" sync / 1[56]:[^:]*:Hydaelyn:(6F11|65C5):/
 2074.9 "Equinox" sync / 1[56]:[^:]*:Hydaelyn:65C3:/
 2082.1 "--top-middle--" sync / 1[56]:[^:]*:Hydaelyn:65A9:/
 2088.6 "Radiant Halo" sync / 1[56]:[^:]*:Hydaelyn:65B4:/
 
-2097.7 "Lightwave" sync / 1[56]:[^:]*:Hydaelyn:6693:/ jump 2206.7
+2097.7 "Lightwave" sync / 1[56]:[^:]*:Hydaelyn:669[345]:/ jump 2206.7
 2117.2 "Echoes x5" #sync / 1[56]:[^:]*:Hydaelyn:65B5:/
 2125.0 "Bright Spectrum" #sync / 1[56]:[^:]*:Hydaelyn:65B9:/
 2125.0 "Dichroic Spectrum" #sync / 1[56]:[^:]*:Hydaelyn:65B8:/
 2137.3 "Crystallize" #sync / 1[56]:[^:]*:Hydaelyn:659A:/
 
+
 # Lightwave 3
-2206.7 "Lightwave" #sync / 1[56]:[^:]*:Hydaelyn:6693:/
+2206.7 "Lightwave" sync / 1[56]:[^:]*:Hydaelyn:669[345]:/
```

Comment:
I always saw the first lightwave as 6693, but I saw others as 669[345], so I left this one as is possibly being any of them, as missing this sync would be bad.  It looks like there's only three values though: https://xivapi.com/search?indexes=action&string=Lightwave&columns=Name,ID&pretty=true

Reply:
Makes sense. When making this timeline, I forgot that the lightwaves have different patterns such as which side will hit first.